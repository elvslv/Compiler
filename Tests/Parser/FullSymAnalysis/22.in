var i, j, n, l, k: integer;
begin
	for i := 1 to 3.5 do
		l := l + k;
end.