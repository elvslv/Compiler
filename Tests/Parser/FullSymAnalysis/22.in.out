var I : INTEGER;
var J : INTEGER;
var K : INTEGER;
var L : INTEGER;
var N : INTEGER;

23 3 Final expression must be integer